src/routes/comic.$comicSlug.tsx

```diff
@@ -0,0 +1,11 @@
+import { createFileRoute } from "@tanstack/react-router";
+import ComicContent from "../components/comic/ComicContent";
+
+export const Route = createFileRoute("/comic/$comicSlug")({
+  component: Comic,
+});
+
+function Comic() {
+  const { comicSlug } = Route.useParams();
```

Comment:
Just check that comicSlug's type is not any, it looks to me that it is. But I might be wrong.

Reply:
Thank you. I will check everything in my next branch.